python/cudf/cudf/core/multiindex.py

```diff
@@ -145,6 +149,10 @@ def __init__(
         name=None,
         verify_integrity=True,
     ):
+        if isinstance(levels, (pd.MultiIndex, MultiIndex)):
```

Comment:
Maybe worth filing an issue, so we ensure this gets fixed?

Reply:
Good idea, opened https://github.com/rapidsai/cudf/issues/18900